drivers/gles3/texture_loader_gles3.cpp

```diff
@@ -48,21 +48,14 @@ RES ResourceFormatGLES2Texture::load(const String &p_path, const String &p_origi
 
 	dstbuff.resize(rowsize * height);
 
-	uint8_t **row_p = memnew_arr(uint8_t *, height);
-
-	for (unsigned int i = 0; i < height; i++) {
-		row_p[i] = 0; //No colors any more, I want them to turn black
-	}
-
-	memdelete_arr(row_p);
```

Comment:
I would leave that code unchanged until @lawnjelly can shed some light on it.
Then we can merge the rest of this PR which would be good as is.